superset-frontend/packages/superset-ui-core/src/chart/components/reactify.tsx

```diff
@@ -17,7 +17,8 @@
  * under the License.
  */
 
-import React from 'react';
+// eslint-disable-next-line no-restricted-syntax -- whole React import is required for `reactify.test.tsx` Jest test passing.
```

Comment:
Appreciate the comments!

Reply:
I should clarify that I was unable to refactor the `reactify.test.tsx` to decouple it from whole React import :(

I hope someone much better than me can do the needful in the future.